### PR TITLE
Visualize dummy pipes

### PR DESCRIPTION
**Pull request by zuckerruebe**
E.g.:

grafik

### CODE REVIEW

**Comment by zuckerruebe** [comment]
We need to deal with duplication

File: trnsysGUI/segments/Node.py [hunk 1]
```diff
@@ -38,38 +38,14 @@ def setParent(self, parent):
...
-        #     return self
-
-        # Should do the same:
         res = self
         while res.nextN() is not None:
             res = res.nextN()
-            # print("In traversal... at node " + str(self) + " has parent " + str(self.parent))
-        return res
-
-    def countNodes(self):
```
Comment (ahobeost) [inline trnsysGUI/segments/Node.py:68]: @zuckerruebe: Is this removal of dead code?

File: trnsysGUI/segments/Node.py [hunk 2]
```diff
@@ -38,38 +38,14 @@ def setParent(self, parent):
...
-        # else:
-        #     return self
-
-        # Should do the same:
         res = self
         while res.nextN() is not None:
             res = res.nextN()
-            # print("In traversal... at node " + str(self) + " has parent " + str(self.parent))
-        return res
-
```
Comment (ahobeost) [inline trnsysGUI/segments/Node.py:67]: @zuckerruebe: Is the following removal of dead code?

File: trnsysGUI/segments/SegmentItemBase.py [hunk 3]
```diff
@@ -90,31 +92,6 @@ def __init__(self, startNode, endNode, parent: "ConnectionBase"):
     def segLength(self):
         return calcDist(self.line().p1(), self.line().p2())
```
[the remark this answers is not in the document]
Comment (ahobeost) [reply trnsysGUI/segments/SegmentItemBase.py:94]: Found it. 
This is pushed down to the singlePipe version, as the doublePipe version doesn't require this

File: trnsysGUI/segments/SegmentItemBase.py [hunk 4]
```diff
@@ -224,9 +196,7 @@ def mouseMoveEvent(self, e):
                 self.logger.debug("Unrecognized editorMode in segmentItem mouseMoveEvent")
 
     def deleteNextHorizSeg(self, b, nextS):
-        if b:
```
Comment (ahobeost) [inline trnsysGUI/segments/SegmentItemBase.py:227]: @zuckerruebe: Any idea what "b" is supposed to stand for?
This occurs further down as well

Comment (ahobeost) [reply trnsysGUI/segments/SegmentItemBase.py:227]: I have changed this to keepSeg and keepPrevSeg to minimize mental effort.

**Comment by zuckerruebe** [comment]
@ahobeost: I hope I've addressed all your points?

**Pull request merged**